cl/simple_pages/templates/help/tags_help.html

```diff
@@ -69,9 +84,52 @@ <h2 id="editing-tags">Editing or Deleting a Tag</h2>
 
 
     <hr>
-    <h2>Coming Soon</h2>
+    <h3>Coming Soon</h3>
     <p>For now, we only allow tagging dockets, but we hope to add more tags to more parts of CourtListener too.
     </p>
     <p>Please stay tuned!</p>
+
+    <h2 id="favorites">Favorites</h2>
+    <p>You can use the Favorites feature to save Dockets, RECAP Documents, Opinions, and Oral arguments within your Favorites list. You can add notes to each favorite item as well.</p>
+    <h3 id="making-favorites">Favorite an item</h3>
+    <p>To favorite an item on CourtListener, you must be logged in and go to the Docket, RECAP Document, Opinion, or Oral Argument that you want to add to your favorites list.</p>
+    <p>Then click on the star icon located to the left of the item title.</p>
+    <p><img src="{% static "png/favorite-title.png" %}"
+         alt="screenshot of the star icon to favorite an item"
+         class="img-responsive img-rounded shadow center-block"
+         height="225"
+         width="557">
+    </p>
+    <p>A modal pops up:</p>
+    <p><img src="{% static "png/make-a-favorite.png" %}"
+         alt="screenshot of the creating a favorite modal"
+         class="img-responsive img-rounded shadow center-block"
+         height="502"
+         width="913">
+    </p>
+    <p>Complete the box with following information:</p>
+    <ol>
+      <li>Name: Required field, by the default the item title, you can change it.</li>
+      <li>Notes: Optional, a note to the favorite, until 500 characters.</li>
+    </ol>
+    <p>Click on the save button.</p>
+    <h3 id="editing-favorites">Viewing, Editing or Deleting a Favorite</h3>
+    <p>You can view your favorite items in the Favorites tab within your user profile panel.</p>
+    <p><img src="{% static "png/view-favorites.png" %}"
+         alt="screenshot of the favorites tab"
+         class="img-responsive img-rounded shadow center-block"
+         height="503"
+         width="1164">
```

Comment:
This width seems really wide. This should be set to the max width of the container.

Reply:
Yeah, I have fixed the with and height accordingly to the max width of the container.